automod/README.md

```diff
@@ -30,27 +30,29 @@ Here is a simple example rule, which handles creation of new events:
 ```golang
 var gtubeString = "XJS*C4JDBQADN1.NSBN3*2IDNEN*GTUBE-STANDARD-ANTI-UBE-TEST-EMAIL*C.34X"
 
-func GtubePostRule(evt *automod.RecordEvent, post *appbsky.FeedPost) error {
+func GtubePostRule(c *automod.RecordContext, post *appbsky.FeedPost) error {
 	if strings.Contains(post.Text, gtubeString) {
-		evt.AddRecordLabel("spam")
+		c.AddRecordLabel("spam")
 	}
 	return nil
 }
 ```
 
 Every new post record will be inspected to see if it contains a static test string. If it does, the label `spam` will be applied to the record itself.
 
-The `evt` parameter provides access to relevant pre-fetched metadata; methods to fetch additional metadata from the network; a `slog` logging interface; and methods to store output decisions. The runtime will catch and recover from unexpected panics, and will log returned errors, but rules are generally expected to run robustly and efficiently, and not have complex control flow needs.
+The `c` parameter provides access to relevant pre-fetched metadata; methods to fetch additional metadata from the network; a `slog` logging interface; and methods to store output decisions. The runtime will catch and recover from unexpected panics, and will log returned errors, but rules are generally expected to run robustly and efficiently, and not have complex control flow needs.
```

Comment:
I'll understand if this doesn't feel worth changing, but I probably would advocate for `ctx` here over just `c`.  I'm not really a fan of single-letter vars (even if it's acclaimedly gopheric).  I think it's also fairly common to reuse the "ctx" identifier among things that have "business context" patterns -- for ex, [urfave CLI](https://pkg.go.dev/github.com/urfave/cli#Command.Run) did this in their docs conventions.

(I see [echo context](https://echo.labstack.com/docs/context) went the other way and kept to a single letter, so it's certainly a matter of preference.)

Reply:
Hmm! We use `cctx` for urfave CLI context, fwiw. (I didn't make that decision, just copied existing code). I think I might find it a bit confusing and would lean towards sticking with `c`? But not a tightly-held opinion if you have stronger feelings.